projects/packages/forms/src/dashboard/components/InboxStatusToggle/index.js

```diff
@@ -66,11 +65,15 @@ export default function InboxStatusToggle( { currentQuery } ) {
 
 	return (
 		<ToggleGroupControl
+			__next40pxDefaultSize
+			__nextHasNoMarginBottom
+			hideLabelFromVision
```

Comment:
We need to add label as well for a11y reason. Not just hide the empty one.

Reply:
Yep, added to line 74. Might've not been clear since I re-ordered all props at the same time 🙃